### PR TITLE
improved version bundle version metrics

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: service/resource/deployment/current.go [hunk 1]
```diff
@@ -49,20 +47,35 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	return currentDeployments, nil
 }
 
-func getVersionBundleVersionInfos(labels map[string]string) (string, string, string, error) {
-	version, ok := labels[VersionBundleVersionLabel]
-	if !ok {
-		return "", "", "", microerror.Maskf(executionFailedError, "label '%s' must not be empty", VersionBundleVersionLabel)
-	}
+func updateVersionBundleVersionGauge(gauge *prometheus.GaugeVec, deployments []*v1beta1.Deployment) error {
```
Comment (JosephSalisbury) [inline service/resource/deployment/current.go]: I wouldn't bother passing the gauge in

Comment (xh3b4sd) [reply service/resource/deployment/current.go]: My thinking was towards testability. It is maybe easier to test the method that way when it does not depend on magic global vars that much.

Comment (JosephSalisbury) [reply service/resource/deployment/current.go]: No strong opinion <£

**Comment by xh3b4sd** [comment]
Successfully tested on lycan.

**Pull request merged**